stdlib/3/os/__init__.pyi

```diff
@@ -3,7 +3,7 @@
 
 # based on http: //docs.python.org/3.2/library/os.html
 
-from builtins import OSError as error
+from builtins import function, OSError as error
```

Comment:
I think we shouldn't use `function` because the type doesn't actually exist in builtins (it's in the stub because mypy needs it). Can you use `Callable[..., Any]` instead?

mypy doesn't think `builtins.function` is compatible with `Callable`.